core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -229,7 +232,12 @@ angular.module('oppia').directive('stateInteractionEditor', [
                         UrlInterpolationService.getInteractionThumbnailImageUrl(
                           interactionId));
                     };
-
+                    $scope.oldToNewListMappingObject = {
```

Comment:
Ditto about naming convention here and in other places.

Reply:
done